generate/definitions/enums

```diff
@@ -119,6 +119,19 @@ TransactionState int8 camelcase (
   7: PrepareEpochFence
 )
 
+////////////
+// QUOTAS //
+////////////
+// QuotasMatchType specifies how to match a Quota entity as part of the DescribeClientQuotasRequestComponent
+QuotasMatchType int8 (
+  // Match on the name exactly
+  0: EXACT
+  // Match on the EntityType's default name
+  1: DEFAULT
+  // Match any specified name
```

Comment:
Perhaps this should be `ANY`? And maybe this doc should be something like,
```
// Returns all named quotas and default quotas for the given EntityType.
```
? (if that's the correct behavior)